src/Ar/Hammers/Hammers.fun

```diff
@@ -166,3 +166,9 @@ END_FUNCTION_BLOCK
 		internal : SdInternal_typ;
 	END_VAR
 END_FUNCTION_BLOCK
+
+FUNCTION SimulationStatus : BOOL (*Used to find the Simulation status for your project*) (*$GROUP=User,$CAT=User,$GROUPICON=User.png,$CATICON=User.png*)
+	VAR_INPUT
+		pOverride :  UDINT; (**)
+	END_VAR
+END_FUNCTION

```

Comment:
I dont get this API. So it returns isSimulation if called with a 0. If called with a BOOL* it will return the dereferenced value of the pointer. Why?

Reply:
The intent of the function is to be a stand alone resource for testing if simulation is active, or it could be setup to be controlled by a different process variable.  In our case, that process variable would be gSimulation.  I've received feedback that in certain cases, it's 'better' to have a global process variable controlling the simulation status for a project.  I'm happy to chat about this if this explanation doesn't answer your question.